azure/folder.go

```diff
@@ -254,3 +245,21 @@ func getUploadStreamToBlockBlobOptions(settings map[string]string) azblob.Upload
 	}
 	return azblob.UploadStreamToBlockBlobOptions{MaxBuffers: maxBuffers, BufferSize: bufferSize}
 }
+
+// Function will get environment's name and return string with the environment's Azure storage account endpoint suffix.
+// Expected names AzureUSGovernmentCloud, AzureChinaCloud, AzureGermanCloud. If any other name is used the func will return
+// the Azure storage account endpoint suffix for AzurePublicCloud.
+func getStorageEndpointSuffix(environmentName string) string {
+	var storageEndpointSuffix string
+	switch environmentName {
+	case "AzureUSGovernmentCloud":
+		storageEndpointSuffix = azure.USGovernmentCloud.StorageEndpointSuffix
+	case "AzureChinaCloud":
+		storageEndpointSuffix = azure.ChinaCloud.StorageEndpointSuffix
+	case "AzureGermanCloud":
```

Comment:
same here

Reply:
See answer in [previous comment](https://github.com/wal-g/storages/pull/43#discussion_r569373883).

Reply:
fixed